languages-and-frameworks/partials/_launch_with_postgres.html.erb

```diff
@@ -3,7 +3,7 @@
 When we run `fly launch` from the newly-created project directory, the launcher will:
 
 * Ask you to select a deployment region
-* Set secrets required by <%= detected %> (`SECRET\_KEY\_BASE`, for example)
+* Set secrets required by <%= detected %> (`SECRET\\_KEY\\_BASE`, for example)
```

Comment:
Here and elsewhere, you can remove the backslashes, since they aren't needed to escape the `_` and `:` characters any more.

Reply:
I removed only them in this file for now.